### PR TITLE
Add tests send message trigger

### PR DESCRIPTION
**Pull request by AurelienFT**
To be merged after #10 and in a tag 18

### CODE REVIEW

File: assembly/massa/message_trigger/main.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { generateEvent, createSC, call, Address, Args, getOpData, toBytes } from "@massalabs/massa-as-sdk";
+
+function createContract(): Address {
+    const bytes: StaticArray<u8> = getOpData(toBytes("smart-contract"));
```
Comment (sydhds) [inline assembly/massa/message_trigger/main.ts:4]: Maybe add a small comment with the path of the retrieved smart contract?

File: assembly/massa/message_trigger/smart-contract.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+/* Age SC Implementation for Massa Labs
```
Comment (sydhds) [inline assembly/massa/message_trigger/smart-contract.ts:1]: How is the comment relevant here?

File: assembly/massa/message_trigger/main.ts (hunk 1 above)
Comment (AurelienFT) [reply assembly/massa/message_trigger/main.ts:5]: Added.

File: assembly/massa/message_trigger/smart-contract.ts (hunk 2 above)
Comment (AurelienFT) [reply assembly/massa/message_trigger/smart-contract.ts]: Oops, I didn't saw it. Ty !

**Review by sydhds — APPROVED** [review]

**Pull request merged**